src/base64.h

```diff
@@ -88,7 +88,7 @@ size_t base64_decode_fast(char* const dst, const size_t dstlen,
                           const size_t decoded_size) {
   const size_t available = dstlen < decoded_size ? dstlen : decoded_size;
   const size_t max_k = available / 3 * 3;
-  size_t max_i = srclen / 4 * 4;
+  size_t max_i = srclen & ~0x03;
```

Comment:
`sizeof size_t` is platform-dependent so that might not work – more reasons to rely on the compiler to figure it out.